exchange/exchange.go

```diff
@@ -179,6 +200,10 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 	return e.buildBidResponse(ctx, liveAdapters, adapterBids, bidRequest, resolvedRequest, adapterExtra, auc, errs)
 }
 
+func (e *exchange) GetCache() prebid_cache_client.Client {
```

Comment:
I don't think you need this. Please see my other comment.

Reply:
Removed. Video endpoint now has a direct reference to the cache.